Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -501,6 +501,17 @@ def _set_up_derived_options(port, options):
             raise RuntimeError('--site-isolation implicitly sets the result flavor, this should not be overridden')
         options.result_report_flavor = 'site-isolation'
 
+    if port.port_name == "ios" and options.site_isolation:
+        host = Host()
```

Comment:
`port` object has a host, should be `port.host.scm()` instead of constructing a new `host` object.